lib/path.js

```diff
@@ -1343,6 +1440,10 @@ const posix = {
 
   format: FunctionPrototypeBind(_format, null, '/'),
 
+  /**
+   * @param {string} path 
+   * @returns {{ dir: string; root: string; base: string; name: string; ext: string }}
```

Comment:
```suggestion
   * @returns {{
   *   dir: string;
   *   root: string;
   *   base: string;
   *   name: string;
   *   ext: string;
   *   }}
```
Max line length -> 80.